doc/ExpertOptions.md

```diff
@@ -1254,33 +1268,23 @@ By default, temporary workspaces hold the results of executed workflows and appl
 }
 ```
 
-This will be passed down through the entire workflow, sub-workflows, and tasks. Workspaces will remain intact for 72 hours. This is a runtime flag, so you will need to run the toplevel workflow with that flag:
+This will be applied to the top-level workflow, sub-workflows, and tasks during compilation.
+
+However, since it is a runtime option, compiling your workflow with this attribute set to true will only guarantee the workspaces of subjobs (that sprawn from the parent job) to remain intact for around 3 days after the analysis. 
```

Comment:
implementation:
https://github.com/dnanexus/nucleus/blob/164c088c9fb7f09490ec7cfbfed9b14df11ce3fa/apiserver/execution/exec_handlers/executions._js#L84-L88
only as a job input argument but not one used when building executables.

Reply:
Yes, thanks! It can be set both on an analysis and a job.